.swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata

```diff
@@ -1,4 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Workspace
    version = "1.0">
+   <FileRef
+      location = "self:">
+   </FileRef>
```

Comment:
This file is now [ignored by default](https://github.com/apple/swift-package-manager/blob/ce50cb0de101c2d9a5742aaf70efc7c21e8f249b/Sources/Workspace/InitPackage.swift#L263) when creating a Swift package with newer versions of SPM, so we might want to go ahead and remove it and add it to the gitignore file.